tests/component-tests/uiOptions/js/UIOptionsTests.js

```diff
@@ -101,6 +109,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 {
                     funcName: "fluid.tests.uiOptions.prefsEditorLocalizedTester.verifySlidingPanelMessages",
                     args: ["{prefsEditor}", "prefsEditor", "Préférences de l'utilisateur"]
+                },
+                {
+                    funcName: "jqUnit.assertEquals",
```

Comment:
Other than verifying the propagation of `defaultLocale` for TOC, it would be nice to also check the loaded TOC message to make sure it matches the locale.